src/dtscalibration/datastore.py

```diff
@@ -3535,6 +3581,17 @@ def create_da_ta2(no, i_splice, direction='fw', chunks=None):
             else:
                 loc = da.from_array(self[st_labeli].data, chunks=memchunk[1:])
 
+            # Make sure variance is of size (no, nt)
```

Comment:
Can't we just use `st_vari = np.broadcast_to(st_vari, (no, nt))`?

Reply:
Good suggestion, makes it a bit simpler!